Liroy/bootstrap.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#Step 1 - Update CentOS 7 system
```

Comment:
Comments are required only to explain a complicated code or the reasoning. It clatters the code, which make it hard to read.